kratos/processes/levelset_convection_process.h

```diff
@@ -351,12 +357,12 @@ class LevelSetConvectionProcess
 
         KRATOS_TRY
 
-        Model& current_model = rBaseModelPart.GetModel();
+        mAuxModelPartName = rBaseModelPart.Name() + "_DistanceConvectionPart";
```

Comment:
this should probably done in the constructor no?

Reply:
yep, thats better. done